src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -60,6 +60,9 @@ public enum CassandraRelevantProperties
     ALLOW_UNSAFE_REPLACE("cassandra.allow_unsafe_replace"),
     ALLOW_UNSAFE_TRANSIENT_CHANGES("cassandra.allow_unsafe_transient_changes"),
     APPROXIMATE_TIME_PRECISION_MS("cassandra.approximate_time_precision_ms", "2"),
+    ASYNC_PROFILER_ADVANCED_MODE("cassandra.async_profiler.advanced_mode", "false"),
+    ASYNC_PROFILER_ENABLED("cassandra.async_profiler.enabled", "true"),
+    ASYNC_PROFILER_OUTPUT_DIRECTORY("cassandra.async_profiler.output_directory", "/var/log/cassandra-profiling"),
```

Comment:
@bbotella  look how it is done in case of e.g. `LOG_DIR_AUDIT`

    /**
     * Directory where Cassandra persists logs from audit logging. 
     * If this property is not set, the audit log framework
     * will set it automatically to 
     * {@link CassandraRelevantProperties#LOG_DIR} + "/audit".
     */
    LOG_DIR_AUDIT("cassandra.logdir.audit"),

and then

    /**
     * Directory where Cassandra puts its logs, defaults to "." which is current directory.
     */
    LOG_DIR("cassandra.logdir", "."),

This should act similarly. The logs for profiling should go to `LOG_DIR + /cassandra-profiling`

Also it should be renamed to `ASYNC_PROFILER_LOG_DIR`.